### PR TITLE
Issue warning...

### PR DESCRIPTION
**Pull request by angerhang**

### CODE REVIEW

File: sty/statements/statements.dtx [hunk 1]
```diff
@@ -1368,8 +1368,10 @@ DefConstructor('\inlinedef OptionalKeyVals:omtext {}', sub {
    my $symbolnode = XML::LibXML::Element->new('symbol');
    $symbolnode->setAttribute(name=>$symb);
    $symbolnode->setAttribute("xml:id"=>makeNCName("$symb.def.sym"));
+ if ($statement_ancestor) {
    $statement_ancestor->parentNode->insertBefore($symbolnode,$statement_ancestor);
- }
+  } else {
+   Error('expected', $statement_ancestor, undef, "'$xc' missing ancestor node.");}}
```
Comment (dginev) [inline sty/statements/statements.dtx]: The human-readable bit is the symbol name ```$symb```. The ```$xc``` XPath context would look completely alien to an sTeX user.

Comment (dginev) [inline sty/statements/statements.dtx]: You could use ```$original_node``` if you need a node, but the important error is missing the ```omdoc:CMP ancestor```, not just an arbitrary ancestor.

**Comment by angerhang** [comment]
Indeed the error message a lot more human-friendly then before : )
I naively thought an ancestor node by convention refers to  `omdoc:CMP ancestor`.

**Comment by angerhang** [comment]
Thanks!

**Pull request merged**